doc/source/timeseries.rst

```diff
@@ -191,6 +187,25 @@ or ``format``, use ``to_datetime`` if these are required.
 
     pd.Timestamp('2010/11/12')
 
+Providing a Format Argument
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In addition to the required datetime string, a ``format`` argument can be passed to ensure specific parsing.
+It will potentially speed up the conversion considerably.
+
+For example:
+
+.. ipython:: python
+
+    pd.to_datetime('2010/11/12', format='%Y/%m/%d')
+
+    pd.to_datetime('12-11-2010 00:00', format='%d-%m-%Y %H:%M')
+
+For further format options see: https://docs.python.org/3/library/datetime.html#strftime-and-strptime-behavior
```

Comment:
can you format the link

`` `text-to-display <linktext>`__ ``

Reply:
I have put the linktext whithin the arrows and pushed. Hope that'll do

Reply:
After going through the contribution docs. I rebased and pushed the branch. Sorry for the inconvenience, that I first pushed to master.